include/fizzy/fizzy_errors.h

```diff
@@ -0,0 +1,22 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2021 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Fizzy error codes.
+/// @file
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum FizzyErrorCode
+{
+    FIZZY_SUCCESS = 0,
+    FIZZY_ERROR_PARSER_INVALID_MODULE_PREFIX,
```

Comment:
How deep we want to go with error reporting?

Reply:
You mean having only 3 codes for parser, validation and instantiate vs a code for each error?

I think a code for each error is useful, but as a first version we could make a simple version with 3 errors, then later go deeper.